docs/wiki/testing/fuzzing.md

```diff
@@ -0,0 +1,40 @@
+# Fuzzing Ethereum's devp2p Protocol
```

Comment:
the title shouldn't be only focused on devp2p but take a general approach of explaining fuzzing testing